docker-compose.yaml

```diff
@@ -3,32 +3,32 @@
 version: "3"
```

Comment:
I believe we can also remove the `version` attribute since it's deprecated.

```suggestion
version: "3"
```

Reply:
Excellent